src/components/Settings/Settings.tsx

```diff
@@ -11,21 +16,17 @@ type Props = {
         selectedAudio: MediaDeviceInfo | false,
         screenOn: boolean
     ) => void;
-    selectedCameraDevice?: MediaStreamTrack;
-    selectedAudioDevice?: MediaStreamTrack;
+    selectedCameraDevice: MediaDeviceInfo | false;
+    selectedAudioDevice: MediaDeviceInfo | false;
```

Comment:
```suggestion
    selectedCameraDevice?: MediaDeviceInfo;
    selectedAudioDevice?: MediaDeviceInfo;
```
we should think about the term `false` here.
the condition that you wrote sounds like:
> selectedCameraDevice is MediaDeviceInfo OR false

which doesn't really seem like a reasonable logic for this variable.
if there is no `MediaDeviceInfo` at all, then is it `undefined` (and not `false`) - that sounds better and logic

Reply:
done